Cargo.toml

```diff
@@ -30,5 +30,5 @@ opt-level = 3
 mio = { git = "https://github.com/servo/mio.git", branch = "servo" }
 # https://github.com/retep998/winapi-rs/pull/816
 winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
-spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu" }
+spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu-servo" }
```

Comment:
Please also change the URL to https://github.com/servo/spirv_cross, which I’ve just forked from kvark/spirv_cross.